lib/govuk/diff/pages/visual_diff/runner.rb

```diff
@@ -0,0 +1,30 @@
+require 'yaml'
+require 'tempfile'
+require 'open-uri'
+
+module Govuk
+  module Diff
+    module Pages
+      module VisualDiff
+        class Runner
+          def initialize(list_of_pages_uri:)
```

Comment:
That's a lot of nested modules we have there! I know it's not new, but could we combine some of these in a future PR?

Eg
```ruby
module GovukDiffPages
  module VisualDiff
    class Runner
    end
  end
end
```